tests/py/requirements.txt

```diff
@@ -10,7 +10,7 @@ torchvision==0.18.0
 pyyaml
 tensorrt==10.0.1
 timm>=1.0.3
-transformers==4.39.3
+transformers==4.40.2
```

Comment:
Approved to match other requirement.

https://github.com/pytorch/TensorRT/blob/c174691727dc78186b7439b404c11d9f4458a4aa/tests/modules/requirements.txt#L1-L2